src/index.html

```diff
@@ -17,16 +17,40 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <img
+        class="icon-big"
+        src="/src/images/Search.svg"
+        alt="search-icon-big"
+      />
```

Comment:
The task specifies that icons should be implemented using the `background-image` CSS property, but an `<img>` tag is used here. Please replace the `<img>` tag with a CSS implementation using `background-image`.